pkg/cmd/render/render.go

```diff
@@ -621,18 +621,16 @@ func writeManifests(outputDir, templateDir string, templateData interface{}) err
 	return nil
 }
 
-func isSingleStackIPv6(serviceCIDRs []string) (bool, error) {
+func preferIPv6(serviceCIDRs []string) (bool, error) {
 	if len(serviceCIDRs) == 0 {
-		return false, fmt.Errorf("isSingleStackIPv6: no serviceCIDRs passed")
+		return false, fmt.Errorf("preferIPv6: no serviceCIDRs passed")
 	}
-	for _, cidr := range serviceCIDRs {
-		ip, _, err := net.ParseCIDR(cidr)
-		if err != nil {
-			return false, err
-		}
-		if ip.To4() != nil {
-			return false, nil
-		}
+	ip, _, err := net.ParseCIDR(serviceCIDRs[0])
```

Comment:
can you please leave a comment here why we do that? 

maybe with a quick example of a serviceCIDR listing and it's desired behavior in terms of bootstrap IPs

Reply:
Sure.

Reply:
perfect, thanks!